scripts/update-tezos.sh

```diff
@@ -24,7 +24,7 @@ cd ..
 rm -rf upstream-repo
 
 git clone --bare https://gitlab.com/tezos/opam-repository.git upstream-opam-repository
-opam_repository_branch="$(git --git-dir=upstream-opam-repository for-each-ref --contains "$opam_repository_tag" --count=1 --sort='-refname' --format="%(refname:lstrip=3)" refs/remotes/origin)"
+opam_repository_branch="$(git --git-dir=upstream-opam-repository for-each-ref --contains "$opam_repository_tag" --count=1 --sort='-refname' --format="%(refname:lstrip=-1)" refs/)"
```

Comment:
given that this is a `bare` git repo on creation there are no branches (except the same one that existed on remote when it was cloned) which I guess is why the `refs/` are the way they are.

In this case there are no local/remote duplicated and the `short` formatting options for `refname` should work as there is no ambiguity:  
```suggestion
opam_repository_branch="$(git --git-dir=upstream-opam-repository for-each-ref --contains "$opam_repository_tag" --count=1 --sort='-refname' --format="%(refname:short)" )"
```